stroyprombeton/models.py

```diff
@@ -343,6 +343,18 @@ class Meta(pages.models.ModelPage.Meta):
     objects = pages.models.ModelPage.create_model_page_managers(Product)
 
 
+class SeriesPage(pages.models.ModelPage):
+    """Proxy model for Admin."""
+
+    class Meta(pages.models.ModelPage.Meta):
+        proxy = True
+        verbose_name = _('serie')
+        verbose_name_plural = _('series')
+
+    objects = pages.models.ModelPage.create_model_page_managers(Series)
```

Comment:
Created https://github.com/fidals/refarm-site/issues/326 to remove this awful method